server/schema/giftcard/res.js

```diff
@@ -24,7 +20,10 @@ const resolvers = {
     },
     async createGiftCard(_, {input}, {loggedInUser}) {
       _authAdmin(loggedInUser)
-      return await GiftCard.create(input).then(giftCard => giftCard)
+      return await GiftCard.create(input).then(async function(giftCard) {
```

Comment:
....then(async function(giftCard)  -> try to use arrow function ?
async(giftCard)=>{}